addons/mail/static/tests/discuss_app/discuss_tests.js

```diff
@@ -1829,8 +1829,7 @@ QUnit.test("Message shows up even if channel data is incomplete", async () => {
         ],
         channel_type: "chat",
     });
-    env.services["bus_service"].forceUpdateChannels();
-    await waitUntilSubscribe();
+    await Promise.all([waitUntilSubscribe(), env.services["bus_service"].forceUpdateChannels()]);
```

Comment:
I don't understand the change, `forceUpdateChannels` isn't `async` so it's strange to put it into `Promise.all`.

Is it intended to work like this?
```js
const prom = waitUntilSubscribe();
env.services["bus_service"].forceUpdateChannels();
await prom;
```

Reply:
Woops, I thought `forceUpdateChannels` was async. Yeah that's what it is supposed to look like. I updated the PR.